### DISCUSSION TITLE
select hive catalog [INTERNAL_ERROR]failed to init reader

### DISCUSSION THREAD

**Discussion by W--7V** [apache/doris#20568] [A - General / Q&A]
创建hive catalog成功，查询报错
1)在hive创建的表
+-----------------------------------------------------------------+
|                         createtab_stmt                          |
+-----------------------------------------------------------------+
| CREATE TABLE `t_hive_p`(                                        |
|   `id` varchar(20),                                             |
|   `name` varchar(20),                                           |
|   `gender` varchar(20),                                         |
|   `age` int,                                                    |
|   `addr` varchar(20))                                           |
| ROW FORMAT SERDE                                                |
|   'org.apache.hadoop.hive.serde2.lazy.LazySimpleSerDe'          |
| STORED AS INPUTFORMAT                                           |
|   'org.apache.hadoop.mapred.TextInputFormat'                    |
| OUTPUTFORMAT                                                    |
|   'org.apache.hadoop.hive.ql.io.HiveIgnoreKeyTextOutputFormat'  |
| LOCATION                                                        |
|   'obs://hive-494e/hive/warehouse/hive_db/t_hive_p'             |
| TBLPROPERTIES (                                                 |
|   'bucketing_version'='2',                                      |
|   'transient_lastDdlTime'='1686129596')                         |
+-----------------------------------------------------------------+

2）在doris 查询hive catalog报错
mysql> select * from t_hive_p;
ERROR 1105 (HY000): errCode = 2, detailMessage = (192.168.2.93)[INTERNAL_ERROR]failed to init reader for file s3://lilei-dev2/wangqiwei/hive/warehouse/hive_db/t_hive/000000_0, err: [INTERNAL_ERROR]init aws s3 client error.

**Comment by yagagagaga** [comment]
what's your Doris version? If you are using Doris 2.0, you can refer to [hive-on-obs](https://doris.apache.org/docs/dev/lakehouse/multi-catalog/hive#hive-on-obs).

**Reply by W--7V** [reply]
doris version is 1.2.3

**Reply by yagagagaga** [reply]
Doris 1.2.x currently does not support hive-on-obs, you can use 2.0.
